pkg/registry/flowcontrol/ensurer/strategy.go

```diff
@@ -126,7 +126,12 @@ func NewSuggestedEnsureStrategy() EnsureStrategy {
 // NewMandatoryEnsureStrategy returns an EnsureStrategy for mandatory config objects
 func NewMandatoryEnsureStrategy() EnsureStrategy {
 	return &strategy{
-		alwaysAutoUpdateSpecFn: func(_ wantAndHave) bool {
+		alwaysAutoUpdateSpecFn: func(wh wantAndHave) bool {
```

Comment:
The name of this field is no longer appropriate, now that it holds a _function_ that makes a decision based on the two particular objects involved.  The word "always" is at best confusing.  Also, the word "auto" is not really helpful either.  I suggest taking both of those words out of the name of this field.